DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/GlobalUsings.cs

```diff
@@ -1,3 +1,4 @@
 global using SOG = Objects.Geometry;
+global using SGIS = Objects.GIS;
```

Comment:
Not sure if I would prefer doing `SOG.GIS.XXX` rather than making a new shorthand for it.

Reply:
But its not `Speckle.Objects.Gis`, its `Speckle.Gis`
Maybe we should have named it `SO.G` and `SO.GIS`
(just a thought provoking suggestion)

Reply:
Oh you are so right... 🙇🏼‍♂️🤦🏼‍♂️ Let's leave it be then.......